### PR TITLE
Documentaion has been updated

### PR DESCRIPTION
**Pull request by ghost**
readme has been updated.
contributing.md and code of conduct has been added

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,4 +2,71 @@
...
+```js
+git clone https://github.com/WrippleFoss/mentorFix-API.git
+```
+
+
+
+
+```js
+cd mentorFix-API
+yarn
```
Comment (abhishek71994) [inline readme.md]: can you also add the use-case of npm? Not a biggie but something that might help.

File: readme.md [hunk 2]
```diff
@@ -2,4 +2,71 @@
...
+cd mentorFix-API
+yarn 
+```
+
+### Run locally
+
+
+```js
+$ cd mentorFix-API
+$ yarn start
```
Comment (abhishek71994) [inline readme.md]: Same here as well please?

File: readme.md [hunk 3]
```diff
@@ -2,4 +2,71 @@
...
+## Get in touch
+
+Join our discord server by clicking on this [link](https://discord.gg/7TahF4D)
+
+## Important Notice
+
+> **MentorFix** has been selected at [GirlScript Summer of Code 2019](https://www.gssoc.tech/)!
+
+![GirlScript](https://cdn-images-1.medium.com/max/600/1*47hUn6EfnP5hZkHslmUsxQ.jpeg)
+
```
Comment (abhishek71994) [inline readme.md:69]: Can you also update this link?
https://teams.railsgirlssummerofcode.org/projects/252-mentorfix
While mentioning that we have been featured in https://railsgirlssummerofcode.org/blog/

**Review by abhishek71994 — CHANGES_REQUESTED** [review]

File: readme.md (hunk 3 above)
Comment (ghost) [reply readme.md:69]: You mean replace girlscript with rails girls summer?
@abhishek71994

Comment (abhishek71994) [reply readme.md:69]: No just add on to that. We don't want to replace our accomplishments, right? 😄

**Comment by ghost** [comment]
@abhishek71994 I have made the changes

**Review by abhishek71994 — APPROVED** [review]
Looks 🍏 . Will merge this into the repo.

**Pull request merged**